tests/unit/providers/vector_io/test_sqlite_vec.py

```diff
@@ -44,38 +44,15 @@ async def sqlite_vec_index(embedding_dimension, tmp_path_factory):
 
 
 @pytest.mark.asyncio
-async def test_add_chunks(sqlite_vec_index, sample_chunks, sample_embeddings):
-    await sqlite_vec_index.add_chunks(sample_chunks, sample_embeddings, batch_size=2)
-    connection = _create_sqlite_connection(sqlite_vec_index.db_path)
-    cur = connection.cursor()
-    cur.execute(f"SELECT COUNT(*) FROM {sqlite_vec_index.metadata_table}")
-    count = cur.fetchone()[0]
-    assert count == len(sample_chunks)
-    cur.close()
-    connection.close()
-
-
-@pytest.mark.asyncio
-async def test_query_chunks_vector(sqlite_vec_index, sample_chunks, sample_embeddings, embedding_dimension):
```

Comment:
Why did we remove this? This tests just the vector search similar to FTS in line 76?

We can later move them to test_vector_io_open_ai_vector_store with search, but as of now these test specific search modes and is good to have in here.

Reply:
removed it here and added it to `tests/unit/providers/vector_io/test_vector_io_openai_vector_stores.py`. I tagged you in the test.